src/analysis/retail/evoker/devastation/modules/abilities/Burnout.tsx

```diff
@@ -0,0 +1,43 @@
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { ApplyBuffEvent, ApplyBuffStackEvent, CastEvent } from 'parser/core/Events';
+import SPELLS from 'common/SPELLS';
+import { isFromBurnout } from '../normalizers/CastLinkNormalizer';
+
+class Burnout extends Analyzer {
+  procs: number = 0;
+  consumedProcs: number = 0;
+
+  constructor(options: Options) {
+    super(options);
+
+    this.addEventListener(
+      Events.applybuff.by(SELECTED_PLAYER).spell(SPELLS.BURNOUT_BUFF),
+      this.onApplyBuff,
+    );
+    this.addEventListener(
+      Events.applybuffstack.by(SELECTED_PLAYER).spell(SPELLS.BURNOUT_BUFF),
+      this.onApplyBuff,
+    );
+
+    this.addEventListener(
+      Events.cast.by(SELECTED_PLAYER).spell([SPELLS.LIVING_FLAME_CAST, SPELLS.LIVING_FLAME_DAMAGE]),
+      this.onLivingFlameCast,
+    );
+  }
+
+  onApplyBuff(event: ApplyBuffStackEvent | ApplyBuffEvent) {
+    this.procs += 1;
+  }
+
+  onLivingFlameCast(event: CastEvent) {
+    if (isFromBurnout(event)) {
+      this.consumedProcs += 1;
+    }
+  }
+
+  get wastedProcs() {
+    return this.procs - this.consumedProcs;
+  }
```

Comment:
You're missing one special case here - buff stacks present at fight end will be counted as wasted. You can catch these by listening for FightEnd event and tallying

Reply:
Nice edge case to know about